dist/index.html

```diff
@@ -3,10 +3,68 @@
   <head>
     <meta charset="UTF-8">
     <title>Fitlit</title>
+    <link rel="styles" href="src/css/styles.css">
+    <!-- pick a font style here -->
   </head>
   <body>
-    <h1>Activity Tracker</h1>
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    <aside class="main-row" id="leftMainRow">
+      <header>Good Morning ${name}</header>
+        <h1>Welcome Back *insert emoji*</h1>
```

Comment:
We can probably insert the emoji here once we agree upon which one we should use.